chore(greenhouse): migrate to vite and improve loading extensions (#543)
cloudoperators/juno: apps/greenhouse/README.md

[![Built with Juno](https://cloudoperators.github.io/juno/built-with-juno.svg)](https://github.com/cloudoperators/juno)

This is the shell app for Project Greenhouse. It is the host for all apps that are part of Greenhouse.

# Usage

## Standalone Mode

To create a static, runnable build, execute the following commands:

```bash
cd apps/greenhouse
npx turbo build:static
```

This will generate an `index.html` file along with the necessary assets in the dist folder. You’ll need to copy a `appProps.json` file containing the required props into the dist folder.

## Development Mode

<!-- add note -->

> [!NOTE]  
> Please make sure to copy the **appProps.template.js** file to **appProps.json** before proceeding.

```bash
cd apps/greenhouse
npx turbo dev
```

### Testing

```bash
cd apps/greenhouse
npx turbo test
```

## App Props

These are the customizable application properties (appProps) that you can define in your appProps.json file:

> - **authIssuerUrl**: `"https://endpoint_url_of_the_openid_provider.com"` (required). Endpoint URL of the OpenID provider.

Author: ?

> - **authIssuerUrl**: `"https://endpoint_url_of_the_openid_provider.com"` (required). Endpoint URL of the OpenID provider.

Author: I don't see there a standard

- **authClientId** (required): `"tbd"`. OIDC client ID.

> - **theme**: `"theme-light"` or `"theme-dark"` (default). Override the default theme.

Reviewer: Would it be ok to use the same text at least like doop and heureka? I would like to keep it consistent. I still have to think how to treat Supernova but I would like to not have more versions for the same.
```suggestion
- **theme** (optional): Overrides the default theme. Acceptable values are `"theme-light"` or `"theme-dark"` (default).
```

> - **embedded**: `true` or `false` (default). Set to true if the app is embedded in another existing app or page. If true, the app will not render a page header/footer and only render content.

Reviewer: same here:
```suggestion
- **embedded** (optional): Set to `true` if the app will be embedded within another app or page. When `true`, the app will not display the header or footer, rendering only the content. Default is `false`.
```
